### PR TITLE
fixed auc metric

### PR DESCRIPTION
**Pull request by rabah-khalek**
Fixing AUC score according to the doc in the binary case: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html

### CODE REVIEW

**Review by AbSsEnT — APPROVED** [review]
Nice job!

**Comment by AbSsEnT** [comment]
Approved

**Comment by mattbit** [comment]
@rabah-khalek can you check that this is ok also in `scanner.performance.metrics.AUC`?

**Review by andreybavt — APPROVED** [review]

**Comment by rabah-khalek** [comment]
Thanks for pointing it out @mattbit, I fixed it in the scan, let's see if it breaks any tests.
